### ISSUE TITLE
Prosperity Tracking

### ISSUE THREAD

**Issue by TrollerCoasterGR** [Lurkars/gloomhavensecretariat#441]
### Is your feature request related to a problem? Please describe.

The prosperity bar is not shown in full.

### Describe the solution you'd like

There should be a scroll back and forth option.

### Describe alternatives you've considered

Or the full prosperity bar should be shown in multiple lines.

**Comment by Lurkars** [comment]
The bar should be scroll-able, just doesn't show a scrollbar.

**Comment by TrollerCoasterGR** [comment]
I have the app installed as an app to my laptop but I can't scroll the bar. I just checked it with my smartphone using a browser and it scrolls, it's true.
Maybe I do something wrong?

**Comment by Lurkars** [comment]
Difficult to say, but it should autoscroll when you change prosperity. So at least you should be able to "click you through the end".

**Comment by rdoll** [comment]
Confirmed that the prosperity levels "slide" left or right to keep your current level on screen. Using latest Chrome with mobile viewport.

Starting with prosperity 1.1: 
<img src="https://github.com/Lurkars/gloomhavensecretariat/assets/7015158/afd44121-fb6b-49cc-8f3d-bd816e0bc3fc" width="300px"/>

Clicking on 3.6 slides 4 into view:
<img src="https://github.com/Lurkars/gloomhavensecretariat/assets/7015158/b7c7a7be-1a7e-42a4-9adc-ca01ad150ef0" width="300px"/>

**Comment by TrollerCoasterGR** [comment]
> Confirmed that the prosperity levels "slide" left or right to keep your current level on screen. Using latest Chrome with mobile viewport.
> 
> Starting with prosperity 1.1: <img alt="" width="300px" src="https://user-images.githubusercontent.com/7015158/285300804-afd44121-fb6b-49cc-8f3d-bd816e0bc3fc.png">
> 
> Clicking on 3.6 slides 4 into view: <img alt="" width="300px" src="https://user-images.githubusercontent.com/7015158/285301008-b7c7a7be-1a7e-42a4-9adc-ca01ad150ef0.png">

Yes if you use a browser. It doesn't work if you install it as an app.

**Comment by TrollerCoasterGR** [comment]
And I don't mean sliding to your current prosperity level. I mean scrolling left or right freely. Our group is almost prosperity lvl.4 and we just wanted to scroll back to the start to count our checkmark.

**Comment by HiddenChar** [comment]
It seems like the prosperity tracker is incorrect for Frosthaven (Using Gloomhaven's prosperity map)

**Comment by Lurkars** [comment]
Added a button to expand the prosperity track in [v0.108.2](https://github.com/Lurkars/gloomhavensecretariat/releases/tag/v0.108.2). Hope that fixes the issue for you!

![Image](https://github.com/user-attachments/assets/9262e873-1abe-4366-b00c-3346d7035343)
